app/views/documents/edit.html.erb

```diff
@@ -17,7 +17,10 @@
     <h2 class="govuk-heading-m"><%= @document.name %></h2>
 
     <p class="govuk-body" style="line-height: 1.6;">
-      If you add one or more numbers to this document, it will be published in the decision notice.
```

Comment:
I guess this is more of a design question, but should this helper text not be "helper text" for the numbers field? https://design-system.service.gov.uk/components/text-input/#hint-text

Reply:
It means it will be repeated multiple times on the page, but I guess this is correct in terms of design principles. I'll update the PR and submit it on the basis that it is the 'right' thing to do and we can discuss with design/product at UAT.